pages/linux/cpupower.md

```diff
@@ -0,0 +1,25 @@
+# cpupower
+
+> Tools regarding CPU power and tuning options.
+> This command is available as part of the `cpupower` package, or on Fedora as part of `kernel-tools`.
+> More information: <https://linux.die.net/man/1/cpupower>.
+
+- List CPUs:
+
+`sudo cpupower --cpu cpulist`
+
+- Print information about all cores:
+
+`sudo cpupower --cpu all info`
```

Comment:
```suggestion
`sudo cpupower --cpu {{all}} info`
```

Reply:
It means user provided value. See [token syntax](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#token-syntax).

Reply:
Ah, I see.
![image](https://user-images.githubusercontent.com/64190512/163085539-6d660a79-9afa-4683-aacb-8d1a7d13246b.png)